source/docs/software/advanced-controls/state-space/state-space-flywheel-walkthrough.rst

```diff
@@ -0,0 +1,204 @@
+State-Space Controller Walkthrough
+==================================
+
+.. note:: Before following this tutorial, readers are recommended to have read an :ref:`docs/software/advanced-controls/state-space/state-space-intro:Introduction to state-space control`.
+
+The goal of this tutorial is to provide "end-to-end" tutorial on implementing a state-space controller for a flywheel.  By following this tutorial, readers will learn how to:
```

Comment:
I'm unsure on the verbiage here. Tutorial is mentioned twice, making the second one somewhat redundant. Perhaps 
```suggestion
The goal of this tutorial is to provide "end-to-end" instructions on implementing a state-space controller for a flywheel.  By following this tutorial, readers will learn how to:
```